config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -880,6 +880,7 @@ def generate_misc():
                    build_cluster="k8s-infra-prow-build",
                    extra_flags=[
                        "--image=cos-cloud/cos-105-17412-156-49",
+                       "--set=spec.networking.networkID=default",
```

Comment:
Just curious, why is this needed?
```suggestion
                       "--set=spec.networking.networkID=default",
```

Reply:
It hides this bug

https://github.com/kubernetes/cloud-provider-gcp/issues/612